Example/Tests/MedStickerEncryptionTests.swift

```diff
@@ -207,8 +207,9 @@ final class MedStickerEncryptionTests: XCTestCase {
 
   private func randomData(count: Int) -> Data {
     var data = Data(count: count)
-    data.withUnsafeMutableBytes {
-      SecRandomCopyBytes(kSecRandomDefault, count, $0)
+    data.withUnsafeMutableBytes { ptr in
+      guard let pointer = ptr.baseAddress?.assumingMemoryBound(to: UnsafeRawBufferPointer.self) else { return }
```

Comment:
I'm seeing a potential issue where we just return and don't generate random bytes, could be quite dangerous.